components/ShadowLayer/examples/ShadowDragSquareExampleViewController.swift

```diff
@@ -46,18 +51,20 @@ class ShadowDragSquareExampleViewController: UIViewController {
     // Elevation of the view is changed to indicate that it has been pressed or released.
     // view.center is changed to follow the touch events.
     if sender.state == .began {
-      self.blueView.elevation = .cardPickedUp
+      self.blueView.shadowLayer.elevation = .cardPickedUp
 
       let selfPoint = sender.location(in: self.view)
       movingViewOffset.x = selfPoint.x - self.blueView.center.x
       movingViewOffset.y = selfPoint.y - self.blueView.center.y
+
```

Comment:
Nit: Remove extra line?

Reply:
This falls under the [vertical whitespace](http://google.github.io/styleguide/objcguide.html#vertical-whitespace) in the style guide as part of separating the logical units of this code into distinct blocks.

Reply:
Ack. I figured since it's ending a conditional block, the extra newline wasn't separating groups of statements.